TombEngine/Objects/TR3/Vehicles/upv.cpp

```diff
@@ -227,7 +227,7 @@ namespace TEN::Entities::Vehicles
 		auto UPV = GetUPVInfo(UPVItem);
 		auto* lara = GetLaraInfo(laraItem);
 
-		auto& ammo = GetAmmo(laraItem, LaraWeaponType::HarpoonGun);
+		auto& ammo = GetAmmo(*lara, LaraWeaponType::HarpoonGun);
```

Comment:
Is this correct? I am not familiar with pointers/references in C++, but what exactly is happening here? Making Lara pointer a reference?

Reply:
I learned that it's better favour references over pointers where possible, and I've been gradually pulling conversions through. It might take a while so there will be some of this funny syntax sitting around while it happens.

Reply:
OK, I got it, thanks!